### PR TITLE
Implement L1 to L2 pipeline with alomobile insurances and cities

### PR DESCRIPTION
**Pull request by ienchia**
# Background

Add pipeline for L1 to L2

# Changes

- add python scripts to help with generating SQL for L1 to L2 process.
- automate the scripts using github action
- add config for alomobile insurances and cities to L2

### CODE REVIEW

File: dags/python_scripts/util.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import os
+import yaml
+
+import sys
+sys.path = sys.path[1:] + sys.path[:1] # Because local airflow.py shadows the module
```
Comment (toileto) [inline dags/python_scripts/util.py:5]: Hmmm, maybe we should change the name of the custom airflow.py

Comment (ienchia) [reply dags/python_scripts/util.py]: @toileto turns out we can avoid this by avoiding direct script call, from `python dags/python_scripts/cli.py` to `python -m dags.python_scripts.cli`. Note that this method requires all internal packages have to be absolute (for example, calling `dags/python_scripts.util.py` from `dags/python_scripts/cli.py` is changed from `import util` to `import .util`).

**Comment by ienchia** [comment]
Do not merge this yet, `gensql` github action should failed because it actually generated an untracked file. Need to find out which github action to detect any untracked file.

**Comment by toileto** [comment]
> Do not merge this yet, `gensql` github action should failed because it actually generated an untracked file. Need to find out which github action to detect any untracked file.

Is this what you're looking for? 
https://github.com/nholden/modified-file-filter-action

**Comment by ienchia** [comment]
I saw that, but prefer if there is a more official account that own the action.

**Review by toileto — APPROVED** [review]

**Pull request merged**